homeassistant/components/shelly/update.py

```diff
@@ -59,7 +59,7 @@ class RestUpdateDescription(RestEntityDescription, UpdateEntityDescription):
 
 REST_UPDATES: Final = {
     "fwupdate": RestUpdateDescription(
-        name="Firmware update",
+        name="Firmware",
```

Comment:
Does shelly have `_attr_has_entity_name = True`? Otherwise you can omit the name and it will use the device class translation, which will become `Firmware`

Reply:
No, `has_entity_name` is `False`.